bin/dquota.py

```diff
@@ -70,50 +70,41 @@ def main():
         client = AccountpageClient(token=opts.options.access_token)
 
         user_id_map = map_uids_to_names()
-        gpfs = GpfsOperations()
         storage = VscStorage()
```

Comment:
best to make a `VscStorageOperator` class in vsc-filesystems that does eg
```
class VscStorageOperator(VscStorage):
   def __init__
      VscStorgae.__init__
      load_storage_operator(...)
  

```

Reply:
all done and updated in https://github.com/hpcugent/vsc-filesystems-quota/pull/19/commits/9526d40bb0abbacd6c467af2c218057da9b76550